packages/api/docs/labels.md

```diff
@@ -1,44 +1,44 @@
 <!-- this doc is generated by ./scripts/docs/labels.mjs -->
 
-  # Labels
-  
-  This document is a reference for the labels used in the SDK.
+# Labels
```

Comment:
Kinda double work, but seems OK to me.
1. Codegen runs
2. On/before commit, run prettier to tidy it up

Reply:
it'd be nice if codegen genned code that passed prettier 😛

Reply:
FWIW codegen does do some prettier, but apparently not well enough.

My feeling is that if it's code we check in and/or ship to npm, it should be formatted. We can improve the codegen CLI to handle this, but for now it doesn't hurt to do it here.